1주차/4673.py

```diff
@@ -0,0 +1,21 @@
+#4673
+
+num_list=[]
+for i in range(1,10001):
+  num_str=str(i)
+  for j in range(len(num_str)):
+    i+= int(num_str[j])
+  num_list.append(i)
+
+a=set(num_list) #set: 리스트 중복 제거
+b=[]
+for i in range(1,10001):
+  b.append(i)
```

Comment:
```suggestion
b=set(range(1,10001))
```
이렇게 range를 set으로 바로 바꿀수도 있어요